lib/typescript/httpclient/payload/index.ts

```diff
@@ -0,0 +1,17 @@
+export * from './ChannelApiPayload';
+export * from './ChannelsPayload';
+export * from './ConnectChannelRequestApiPayload';
+export * from './ConnectChannelRequestPayload';
+export * from './ConversationPayload';
```

Comment:
If I understand how the code is organised now, we could export only Request/Response payloads here and we probably should?